### DISCUSSION TITLE
请大佬看看这是什么问题，启动时提示“error!bin path is a dir”

### DISCUSSION THREAD

**Discussion by lexuefeng** [AdguardTeam/AdGuardHome#3145] [Q&A]
openwrt 软路由编译进AdGuardHome无法进行更新，核心版本:core error 没有配置文件no core没有核心，点启动时提示“error!bin path is a dir”请各位帮帮忙解决下。

**Comment by ameshkov** [comment]
嘿，你怎么配置 AdGuard Home？你用标准程序 （install.sh）还是其他的办法？

这个错误的描述看起来是配置错误。

**Comment by lexuefeng** [comment]
您好，我是openwrt里编译进去的AdGuard Home,无法进行更新，无法启动，请帮助看看那里出现问题。谢谢。







在 2021-05-14 22:27:59，"Andrey Meshkov" ***@***.***> 写道：

嘿，你怎么配置 AdGuard Home？你用标准程序 （install.sh）还是其他的办法？

这个错误的描述看起来是配置错误。

**Reply by ameshkov** [reply]
编译啊？你用什么命令编译 AdGuard Home？make？

请显示编译的全日志

**Comment by lexuefeng** [comment]
不好意思，openwrt X86_64bit系统里默认编译完执行文件路径是/usr/bin/AdGuardHome,需要改成/usr/bin/AdGuardHome/AdGuardHome,再应用、启动就不会有问题了。谢谢您的回复。



在 2021-05-17 03:52:09，"Andrey Meshkov" ***@***.***> 写道：

编译啊？你用什么命令编译 AdGuard Home？make？

请显示编译的全日志
